.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Interoperability tests with GnuTLS and NSS
+on:
+  schedule:
+    - cron: '0 6 * * *'
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: docker.io/fedora:39
+      options: --sysctl net.ipv6.conf.lo.disable_ipv6=0
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [gnutls, nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
```

Comment:
Just looking at how much this setup has deviated from the initial one. You might want to remove this (inherited) reference as, at this point, it seems to me quite irrelevant/unhelpful so you can keep this code nice and clean.
Thank you everyone for all the work.